dpctl/_sycl_queue_manager.pyx

```diff
@@ -213,7 +213,7 @@ cpdef get_current_backend():
 @contextmanager
 def device_context(arg):
     """
-    Yields a SYCL queue corresponding to the input filter string.
+    Yields a SYCL queue corresponding to the input device object, the queue object, or the filter string.
```

Comment:
```suggestion
    Yields a SYCL queue corresponding to the input device object, queue object, or filter string.
```

Reply:
Reordered enumeration of argument types: explicit queue instance, explicit device instance, device filter selector string.